pywebostv/controls.py

```diff
@@ -176,6 +181,20 @@ class MediaControl(WebOSControlBase):
         "stop": {"uri": "ssap://media.controls/stop"},
         "rewind": {"uri": "ssap://media.controls/rewind"},
         "fast_forward": {"uri": "ssap://media.controls/fastForward"},
+        "get_audio_output": {
```

Comment:
Can you add this to the README.md as well?

Reply:
done :)